pages/support/index.js

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import { NextSeo } from "next-seo";
+import { getDocsListBySection, getNavigation } from "~/services";
+import { Box, Icon, theme, styled } from "@washingtonpost/wpds-ui-kit";
+import ChevronRight from "@washingtonpost/wpds-assets/asset/chevron-right";
+import { Header } from "~/components/Markdown/Components/headers";
+import Link from "~/components/Markdown/Components/link";
+import { P } from "~/components/Markdown/Styling";
+
+const Masonry = styled("section", {
```

Comment:
I noticed that we're using grids all over the place (so good!) However, I'm thinking that maybe, since we use them all the time, we could pull them out to a utils or libs or something? 

We could do something like: 
```const GridThreeColumns = styled("div", {
    display: "grid",
    gap: "$100",
    "@notSm": {
      gridTemplateColumns: "repeat(3,1fr)",
    },
    "@sm": {
      gridTemplateColumns: "1fr",
    },
  });

  const GridTwoColumns = styled("div", {
    display: "grid",
    gap: "$200",
    "@notSm": {
      gridTemplateColumns: "repeat(2,1fr)",
    },
    "@sm": {
      gap: "$100",
      gridTemplateColumns: "1fr",
    },
  });``` 

or create a variant like you did earlier (with descriptive names like these). Thoughts?

Reply:
love this!